core/rfb.js

```diff
@@ -47,6 +47,7 @@ export default class RFB extends EventTargetMixin {
         this._rfb_credentials = options.credentials || {};
         this._shared = 'shared' in options ? !!options.shared : true;
         this._repeaterID = options.repeaterID || '';
+        this._showDot = options.showDot || true;  // temporarily for debugging
```

Comment:
Try to not push your debugging code as it might be overlooked and actually committed. :)